domain/src/test/java/org/oppia/domain/exploration/ExplorationProgressControllerTest.kt

```diff
@@ -100,6 +102,12 @@ class ExplorationProgressControllerTest {
   @Mock
   lateinit var mockAsyncAnswerOutcomeObserver: Observer<AsyncResult<AnswerOutcome>>
 
+  @Mock
+  lateinit var mockAsyncHintObserver: Observer<AsyncResult<Hint>>
```

Comment:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
Done.